backend/src/test/java/dev/tripdraw/application/MemberServiceTest.java

```diff
@@ -50,4 +53,22 @@ void setUp() {
                 .isInstanceOf(MemberException.class)
                 .hasMessage(NICKNAME_CONFLICT.getMessage());
     }
+
+    @Test
+    void 사용자를_조회한다() {
+        // expect
+        MemberSearchResponse result = memberService.findById(member.id());
+        assertThat(result.nickname()).isEqualTo("통후추");
+    }
+
```

Comment:
혹시 when 절에 속하지 않을까요?
```suggestion
    @Test
    void 사용자를_조회한다() {
        // when
        MemberSearchResponse result = memberService.findById(member.id());

        // expect
        assertThat(result.nickname()).isEqualTo("통후추");
    }

```

Reply:
given을 붙이기 좀 그래서 바로 조회 후 검증을 진행했습니다! 이 부분은 추후에 수정하겠습니다 👍